bitnami/common/templates/_notestxt.tpl

```diff
@@ -0,0 +1,9 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Print instructions to get a secret value.
+Usage:
+{{ include "common.notetxt.secret.getvalue" (dict "varname" "MY_ENVVAR" "secret" "secret-name" "field" "secret-value-field" "context" $) }}
+*/}}
+{{- define "common.notetxt.secret.getvalue" -}}
+export {{ .varname }}=$(kubectl get secret --namespace {{ .context.Release.Namespace }} {{ .secret }} -o jsonpath="{.data.{{ .field }}}" | base64 --decode)
+{{- end -}}
```

Comment:
Shouldn't you use this in PostgreSQL's NOTES.txt?

Reply:
done!